tools/benchmark-core/benches/vnode.rs

```diff
@@ -0,0 +1,27 @@
+use yew::prelude::*;
+
+fn main() {
+    divan::main();
+}
+
+#[function_component]
+fn Stuff(_: &()) -> Html {
+    html! {
+        <p>{"A custom component"}</p>
+    }
+}
+
+#[divan::bench(sample_size = 10000000)]
+fn vnode_clone(bencher: divan::Bencher) {
+    let html = html! {
+        <div class={classes!("hello-world")}>
+            <span>{"Hello"}</span>
+            <strong style="color:red">{"World"}</strong>
+            <Stuff />
+        </div>
```

Comment:
We clone only VTag and VComp here. Probably wanna benchmark the others too, especially VList

Reply:
Pretty sure VList is in there because there is a list of nodes.

I'm also cloning classes and style (which are also very important imo).

(and yes the `<Stuff/>` thingy is purely to get a VComp)

Reply:
Oh, right. It's still missing a few cases though: VSuspense and VRef. Not sure if we can benchmark those

Reply:
@cecton can we benchmark those or should we just merge it in as-is?

Reply:
I could add VSuspense but not VRef